src/components/organisms/PaneManager/PaneManager.tsx

```diff
@@ -199,62 +202,65 @@ const PaneManager = () => {
         </Space>
       </StyledColumnLeftMenu>
 
-      {activeProject ? (
-        <StyledColumnPanes style={{width: contentWidth}}>
-          <SplitView
-            contentWidth={contentWidth}
-            left={
-              <>
-                <div style={{display: leftMenuSelection === 'file-explorer' ? 'inline' : 'none'}}>
-                  <FileTreePane />
-                </div>
-                <div style={{display: leftMenuSelection === 'kustomize-pane' ? 'inline' : 'none'}}>
-                  <KustomizePane />
-                </div>
-                <div style={{display: leftMenuSelection === 'helm-pane' ? 'inline' : 'none'}}>
-                  <HelmPane />
-                </div>
-                <div
-                  style={{
-                    display: leftMenuSelection === 'templates-pane' ? 'inline' : 'none',
-                  }}
-                >
-                  <TemplateManagerPane />
+      {
+        isProjectLoading ? (<Skeleton />) :
```

Comment:
Too many ternary - I think it makes code reading harder

Reply:
true, tried to fix it